airflow/example_dags/example_sla_dag.py

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import time
+from datetime import datetime, timedelta
+
+from airflow.decorators import dag, task
+
+"""Example DAG demonstrating SLA use in Tasks"""
+
+
+# [START howto_task_sla]
+def sla_callback(*args):
+    arguments = ["dag", "task_list", "blocking_task_list", "slas", "blocking_tis"]
+    print(*zip(arguments, args))
+
+
+@dag(
+    "example_sla_dag",
```

Comment:
Including the `dag_id` in the decorator isn't necessary since the `dag_id` will default to the function name being decorated. No strong opinion here though. It's more of a stylistic comment.

Reply:
You're right. I'll remove that